### PR TITLE
bump(*):

### PR DESCRIPTION
**Pull request by adambkaplan**
* Pin s2i to release-3.10 to exclude `--as-dockerfile` feature
* Merge in fix to check assemble-user against allowed UIDS

Bug [1582976](https://bugzilla.redhat.com/show_bug.cgi?id=1582976)
Addresses [CVE-2018-10843](https://access.redhat.com/security/cve/cve-2018-10843)

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -47,6 +47,8 @@ import:
   version: master
 - package: github.com/openshift/service-serving-cert-signer
   version: release-3.10
+- package: github.com/openshift/source-to-image
+  version: release-3.10
```
Comment (adambkaplan) [inline glide.yaml]: Pin here is to exclude the `--as-dockerfile` feature @bparees recently added. This is still experimental and we don't want this in Origin until 3.11

File: vendor/github.com/opencontainers/runc/libcontainer/cgroups/systemd/apply_systemd.go [hunk 2]
```diff
@@ -271,19 +270,13 @@ func (m *Manager) Apply(pid int) error {
...
-		if c.Resources.CpuQuota > 0 {
-			// systemd converts CPUQuotaPerSecUSec (microseconds per CPU second) to CPUQuota
-			// (integer percentage of CPU) internally.  This means that if a fractional percent of
-			// CPU is indicated by Resources.CpuQuota, we need to round up to the nearest
-			// 10ms (1% of a second) such that child cgroups can set the cpu.cfs_quota_us they expect.
-			cpuQuotaPerSecUSec = uint64(c.Resources.CpuQuota*1000000) / c.Resources.CpuPeriod
-			if cpuQuotaPerSecUSec%10000 != 0 {
-				cpuQuotaPerSecUSec = ((cpuQuotaPerSecUSec / 10000) + 1) * 10000
-			}
+		cpuQuotaPerSecUSec := uint64(c.Resources.CpuQuota*1000000) / c.Resources.CpuPeriod
```
Comment (deads2k) [inline vendor/github.com/opencontainers/runc/libcontainer/cgroups/systemd/apply_systemd.go]: This appears to be another unregistered fork.

**Comment by adambkaplan** [comment]
Duplicates #19888

**Pull request closed without merging**